docs/cli/graphql-transformer/auth.md

```diff
@@ -145,20 +145,20 @@ type Todo @model
 }
 ```
 
-In this schema, only the owner of the object has the authorization to perform update (`updateTodo`) and delete (`deleteTodo`) operations on the owner created object, but anyone can read them (`getTodo`, `listTodos`). This prevents the object from being updated or deleted by users other than the creator of the object while allowing all authenticated users of the app to read them.
+In this schema, only the owner of the object has the authorization to perform update (`updateTodo`), create (`createTodo`) and delete (`deleteTodo`) operations on the owner created object, but anyone can read them (`getTodo`, `listTodos`). This prevents the object from being updated or deleted by users other than the creator of the object while allowing all authenticated users of the app to read them.
```

Comment:
I think this is the confusion I've always had with this table in the past. Basically, before create, there is no owner. I wonder if we can convey this better, because in reality anyone can create a todo because there is no owner before creation.

Reply:
1.  I suggest that we change the code snippet to order the operations in the following manner (which correspond to CRUD)
```
operations: [create, update, delete]
```
2.  The wording can then change to something like:
```
In this schema, only the owner of the object has authorization to perform create (`createTodo`), update (`updateTodo`), and delete (`deleteTodo`) on the owner created object, while anyone else can only read (`getTodo`, `listTodos`)
```

Reply:
Despite wanting to change the wording, I think this change is a step in the right direction, so, LGTM!

Reply:
addressed this feedback and made the changes